frontend/plan/src/components/selector/CourseDetails.js

```diff
@@ -5,11 +5,22 @@ import TagList from "./TagList";
 import Badge from "../Badge";
 
 const getReqCode = (school, name) => `${{ SAS: "C", SEAS: "E", WH: "W" }[school]}: ${name}`;
+const getPrereqCodes = (text) => {
+    if (typeof text !== "string") return [];
+    const captured = text.match(/(^|\W)[a-zA-Z]{3}[a-zA-Z]?(-|\s)[0-9]{3}($|(?=\W))/gm);
+    return captured ? captured.map(prereq => prereq.replace(/\W/g, " ").trim()) : [];
```

Comment:
Might be better to show the prereq string by default if the regex doesn't match, because it can still have useful info to students.

Reply:
Talked to Jess, and we're going to display the raw text but annotate the course codes as links